test/test_fx.py

```diff
@@ -2595,23 +2595,24 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
                            '__rpow__'}
 
         try:
-            sample_inputs_itr = op.sample_inputs(device, dtype, requires_grad=False)
+            sample_ctxs = op.sample_inputs(device, dtype, requires_grad=False)
             schemas = get_signature_for_torch_op(op.op)
             if not schemas:
                 raise RuntimeError('No Schemas Returned')
-            for sample_input in sample_inputs_itr:
-                # Iterate through overloads until we hit a match. If we exit this
-                # loop via `else`, we haven't found a match
-                for schema in schemas:
-                    try:
-                        bound_args = schema.bind(sample_input.input, *sample_input.args, **sample_input.kwargs)
-                        bound_args.apply_defaults()
-                        op(*bound_args.args, **bound_args.kwargs)
-                        break
-                    except TypeError as e:
-                        pass
-                else:
-                    raise RuntimeError(f'Did not match any schemas for op {op.name}!')
+            for ctx in sample_ctxs:
```

Comment:
fyi @Chillee @jamesr66a

Reply:
I've suggested not updating fx code in this PR - so these changes will probably be mooted -- never mind the fyi

Reply:
if I revert these changes and we keep the `ctx` API, these tests will start to fail. Do you mean that I should just do that?